Exception ex = new Exception("bu testdir.");


throw ex;